src/lib/containers/UserCardList.tsx

```diff
@@ -93,13 +96,13 @@ export default class UserCardList extends React.Component<
     const nullOwnerIdsRows = data.queryResult!.queryResults.rows.filter(
       el => !el.values[ownerId],
     )
-    return nullOwnerIdsRows.map<UserProfile>(el => {
+    return nullOwnerIdsRows.map<Omit<UserProfile, 'ownerId'>>(el => {
       const values = el.values
       return {
         firstName: values[firstNameIndex] ?? '',
         lastName: values[lastNameIndex] ?? '',
         company: values[institutionIndex] ?? undefined,
-        ownerId: '',
+        ownerId: null,
```

Comment:
If we make this the empty string, the UserCard component will try to fetch the profile, which emits all kinds of errors and warnings to the console. It should really be null.